src/train_model.py

```diff
@@ -69,28 +79,48 @@ def mlp_config():
                      if training_run_string != "" else "")
     save_path_prefix = (model_dir + net_type + '_' + dataset +
                         cluster_gradient * '_clust-grad' + save_path_end)
-    # TODO: figure out what info should go in here.
+
+    # Simple Network config
```

Comment:
IMO this could be a named_config instead.

Reply:
Done, though it gave me errors unless I defined empty versions of each variable in the main config.